src/App.css

```diff
@@ -0,0 +1,40 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
```

Comment:
Nice minimalist style webpage, will look great once all the repo data is rendered!